.claude/agents/test-runner.md

```
@@ -0,0 +1,123 @@
+テスト完了後、以下の形式でメインエージェントに報告してください：
+
+### 全テスト成功の場合
+
+```
+テスト完了: 全 [X] テストが成功しました。
+レポート: reports/tests/[ファイル名].md
+```
+
+### 失敗がある場合
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)